sdk/eventhub/event-hubs/samples/browserSample.html

```diff
@@ -0,0 +1,37 @@
+<!-- 
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT Licence.
+  
+This sample demonstrates a simple send message operation.  
```

Comment:
`message` -> `event`

We are using the term `event` in event hubs in most samples